GliaWidgets/Sources/Theme/Alert/AlertConfiguration.swift

```diff
@@ -48,6 +48,9 @@ public struct AlertConfiguration {
     /// Configuration of the unavailable message center error alert.
     public var unavailableMessageCenter: MessageAlertConfiguration
 
+    /// Configuration of the unavailable message center error alert due to not autheticated visitor.
```

Comment:
due to unauthenticated visitor

Reply:
Updated fafddff891824bea8a0e797368f3e708974faf02.